### DISCUSSION TITLE
Pornographic Stimuli Database

### DISCUSSION THREAD

**Discussion by DominiqueMakowski** [RealityBending/FictionEro#4] [Ideas]
I made a quick search again and it's quite baffling that there is no good and recent validated database of pornographic (/erotic +++) stimuli! 

With AI developments, maybe it could facilitate the creation and validation of a database...
Like for instance creating various AI images matching a wide range of desired qualities (e.g., amateurness, hardcority, aesthetic-appeal/art-likeness, ...), then in the validation phase we intermix them with real images, and we validate them on a real/fake judgment paradigm (on top of all the appraisal scales)

Validating a stim database is a huge endeavour, but with a good team anything is possible... 🙃

**Comment by MarcoViola88** [comment]
Intriguing idea... Let me list some PROs and CONs

**CONS**
1) Not sure that's just in Italy, but working TOO much on sexual stimuli might be risky -- many colleagues are more bigot than they'd admit, and I've been surmounted by a lot of silly jokes already for our previous work. Not enough to scare me away, but enough to make me think "I need a good ratioinale and a solid project to do this, otherwise might not be worth it"
2) What is the legal framework about deepfakes, especially when they depict explicit content? And what will it be in 2 years, given that the international legal framework might be shaky?
3) Even before collection, finding the right photos + producing the right deepfakes might be quite an effort, and we'd definitely need to read some literature on sex research... (but see Pro-2)

**PROs:**
1) If you do this right, that's a lot of citation
2) May be a good thing to write a grant about?

**Comment by DominiqueMakowski** [comment]
> and I've been surmounted by a lot of silly jokes already for our previous work

Oh wow that sucks :/

**Comment by MarcoViola88** [comment]
> > and I've been surmounted by a lot of silly jokes already for our previous work
> 
> Oh wow that sucks :/

Well, maybe not "a lot" -- but still, even few jokes alerted me that many colleagues are not as open-minded as they pretend to me, hence we need to proceed with caution.
In any case, our previous paper taught us that this topic triggers all sorts of all reactions (gotta tell you about our failed attempts to recruit adult content creators as volunteers one day ... so frustrating...). Hence, at the very least, developing and preserving a ZEN attitude should be paramount
